charts/telepresence/templates/trafficManagerRbac/namespace-scope.yaml

```diff
@@ -28,16 +28,25 @@ rules:
   - list
   - get
   - watch
-# Needed for the `gather-logs` command to work on components in the kubernetes
-# cluster (traffic-manager + traffic-agents)
 - apiGroups:
   - ""
   resources:
-  - pods/log
+  - services
   verbs:
   - list
-  - get
 {{- if eq . (include "telepresence.namespace" $) }}
+# Must be able to get the manager namespace in order to get the cluster-id
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  resourceNames:
+  - {{ . }}
+  verbs:
+  - get
```

Comment:
wow im very surprised that giving get namespaces permissions at the namespace scope works, but i guess it makes sense to get the namespace the rolebinding is in